### ISSUE TITLE
File Error: can't sync to MPEG frame

### ISSUE THREAD

**Issue by starenka** [Nicotine-Plus/nicotine-plus#50]
```
08:08:12 Scanning File Error: can't sync to MPEG frame Path: ****
08:08:12 Traceback: HeaderNotFoundError: can't sync to MPEG frame
08:08:12   File "/usr/lib/python2.7/dist-packages/pynicotine/shares.py", line 568, in getFileInfo
    info = metadata.detect(pathname)
08:08:12   File "/usr/lib/python2.7/dist-packages/pynicotine/metadata_mutagen.py", line 19, in detect
    audio = mutagen.File(path)
08:08:12   File "/usr/lib/python2.7/dist-packages/mutagen/_util.py", line 131, in wrapper_func
    return func(h, *args, **kwargs)
08:08:12   File "/usr/lib/python2.7/dist-packages/mutagen/_file.py", line 295, in File
    return Kind(fileobj, filename=filething.filename)
08:08:12   File "/usr/lib/python2.7/dist-packages/mutagen/_file.py", line 48, in __init__
    self.load(*args, **kwargs)
08:08:12   File "/usr/lib/python2.7/dist-packages/mutagen/_util.py", line 123, in wrapper
    return func(self, h, *args, **kwargs)
08:08:12   File "/usr/lib/python2.7/dist-packages/mutagen/id3/_file.py", line 406, in load
    self.info = self._Info(fileobj, offset)
08:08:12   File "/usr/lib/python2.7/dist-packages/mutagen/_util.py", line 152, in wrapper
    return func(*args, **kwargs)
08:08:12   File "/usr/lib/python2.7/dist-packages/mutagen/mp3/__init__.py", line 387, in __init__
    raise HeaderNotFoundError("can't sync to MPEG frame")

```
Debian testing
Package: nicotine                        
Version: 1.2.16+dfsg-1.1

**Comment by gfarmerfr** [comment]
Hi @starenka,

This issue (https://github.com/quodlibet/mutagen/issues/268) is fixed by mutagen 1.36.2 release.
It was affecting mutagen releases from 1.33 to 1.36.2 from what I can see.

From what i can see debian testing is using 1.36.1.

So either you wait for debian to update the package to 1.36.2 or you can use pip to update it yourself.

**Comment by starenka** [comment]
ok. cheers
